src/test/java/io/weaviate/integration/client/batch/ClientBatchMultiTenancyTest.java

```diff
@@ -100,28 +104,28 @@ public void shouldCreateObjects() {
         .id(WeaviateTestGenerics.PIZZA_QUATTRO_FORMAGGI_ID)
         .className("Pizza")
         .properties(propsQuatroFormaggi)
-        .tenant(tenant)
+        .tenant(tenant.getName())
```

Comment:
is this a breaking change?

Reply:
nope. those are only changes in test fixtures.